centreon/tests/e2e/features/Cloud-notifications/02-notification-duplicate/index.ts

```diff
@@ -0,0 +1,153 @@
+    method: 'GET',
+    url: '/centreon/api/latest/configuration/notifications?page=1&limit=10*'
+  }).as('getNotifications');
+
+  cy.addHostGroup({
+    name: data.hostGroups.hostGroup1.name
+  });
+
+  cy.addHost({
+    activeCheckEnabled: false,
+    checkCommand: 'check_centreon_cpu',
+    hostGroup: data.hostGroups.hostGroup1.name,
+    name: data.hosts.host1.name,
+    template: 'generic-host'
+  })
+    .addService({
+      activeCheckEnabled: false,
+      host: data.hosts.host1.name,
+      maxCheckAttempts: 1,
+      name: data.services.service1.name,
+      template: 'Ping-LAN'
+    })
+    .applyPollerConfiguration();
+
+  checkHostsAreMonitored([
+    {
+      name: data.hosts.host1.name
+    }
+  ]);
+
+  checkServicesAreMonitored([
+    {
+      name: data.services.service1.name
+    }
+  ]);
+});
+
+afterEach(() => {
+  cy.stopWebContainer();
+});
+
+Given('a user with access to the Notification Rules page', () => {
+  cy.navigateTo({
+    page: 'Notifications',
+    rootItemNumber: 3,
+    subMenu: 'Notifications'
+  });
+});
+
+Given('the user is on the Notification Rules page', () => {
+  cy.url().should('include', '/configuration/notifications');
+});
+
+Given('a Notification Rule is already created', () => {
+  createNotification(notificationBody);
+});
+
+When('the user selects the duplication action on a Notification Rule', () => {
+  cy.get('#Duplicate').click();
+});
+
+When('the user enters a new Notification Rule name', () => {
+  cy.get('#Newnotificationname').type(duplicatedNotificationName);
+});
+
+When('the user confirms to duplicate', () => {
+  cy.get('[data-testid="Confirm"]').click();
+});
+
+Then('a success message is displayed', () => {
+  cy.get('.MuiAlert-message').should('have.text', 'Notification duplicated');
+});
+
+Then(
+  'the duplicated Notification Rule with same properties is displayed in the listing',
+  () => {
+    cy.wait('@getNotifications');
+    cy.contains(duplicatedNotificationName).should('exist');
+  }
+);
+
+Then(
+  'the duplicated Notification Rule features the same properties as the initial Notification Rule',
+  () => {
+    cy.request({
+      method: 'GET',
+      url: 'centreon/api/latest/configuration/notifications'
+    }).then((response) => {
+      const duplicatedNotification = response.body.result.find(
+        (notification) => notification.name === duplicatedNotificationName
+      );
+      const originalNotification = response.body.result.find(
+        (notification) => notification.name === notificationBody.name
+      );
+
+      notificationProperties.forEach((property) => {
+        expect(duplicatedNotification[property]).to.deep.equal(
+          originalNotification[property]
+        );
+      });
+    });
+  }
+);
+
+When('the user clicks on the discard action', () => {
+  cy.get('[data-testid="Cancel"]').click();
+});
+
+Then('the duplication action is cancelled', () => {
+  cy.get('.MuiTableRow-root.css-1b36c9s-row').should('have.length', 1);
```

Comment:
you should find an other selector because thise hash (`1b36c9s``) will change when there will be frontend changes